pages/docs/reference/packages.md

```diff
@@ -10,20 +10,22 @@ title: "Packages and Imports"
 A source file may start with a package declaration:
 
 <div class="sample" markdown="1" theme="idea" data-highlight-only>
+
 ```kotlin
-package foo.bar
+package org.example
 
-fun baz() { ... }
-class Goo { ... }
+fun printMessage() { /*...*/ }
+class Message { /*...*/ }
 
 // ...
 ```
 </div>
 
 All the contents (such as classes and functions) of the source file are contained by the package declared.
-So, in the example above, the full name of `baz()` is `foo.bar.baz`, and the full name of `Goo` is `foo.bar.Goo`. 
+So, in the example above, the full name of `printMessage()` is `org.example.printMessage`,
+and the full name of `Message` is `org.example.Message`. 
  
-If the package is not specified, the contents of such a file belong to "default" package that has no name.
+If the package is not specified, the contents of such a file belong to `default` package that has no name.
```

Comment:
This change is misleading because `default` looks like an identifier which it is not